src/components/List.jsx

```diff
@@ -1,11 +1,8 @@
+import { shallowEqual, useSelector } from 'react-redux';
 import Book from './Book';
 
 const List = () => {
-  const books = [
-    {
-      id: 1, title: 'Crime and Punishment', author: 'Fyodor Dostoevski', genre: 'Philosophical novel',
-    },
-  ];
+  const books = useSelector((state) => state.books, shallowEqual);
   return books.map((book) => (
     <Book genre={book.genre} title={book.title} author={book.author} key={book.id} />
   ));
```

Comment:
- [ ] Currently nothing is being displayed on your page on initial render, this is because you have not added some error boundaries. try adding ``if (!books) {
    return (
      <h2>Loading</h2>
    );
  }`` before `return books.map` method, this will help display a text Loading when nothing is in the book array.
  this is how your page looks currently 👇 
    
![image](https://user-images.githubusercontent.com/66097371/161140632-c9b0a030-d80b-4fe3-9a5e-db85d5be3426.png)